### PR TITLE
Add ids length constants

### PR DESCRIPTION
**Pull request by StephenButtolph**
## Why this should be merged

Rather than relying on the `hashing` package constants - it's more user friendly to expose the constants from the `ids` package.

## How this works

Move constants.

## How this was tested

N/A

### CODE REVIEW

**Review by joshua-kim — APPROVED** [review]

File: ids/id.go [hunk 1]
```diff
@@ -15,7 +15,10 @@ import (
 	"github.com/ava-labs/avalanchego/utils/wrappers"
 )
 
-const nullStr = "null"
+const (
+	IDLen   = 32
+	nullStr = "null"
```
Comment (abi87) [inline ids/id.go:20]: what is nullStr for?

File: ids/node_id.go [hunk 2]
```diff
@@ -13,7 +13,10 @@ import (
 	"github.com/ava-labs/avalanchego/utils/hashing"
 )
 
-const NodeIDPrefix = "NodeID-"
+const (
+	NodeIDPrefix = "NodeID-"
+	NodeIDLen    = ShortIDLen
```
Comment (abi87) [inline ids/node_id.go:18]: what is NodeIDPrefix for?

File: ids/id.go (hunk 1 above)
Comment (StephenButtolph) [reply ids/id.go:20]: json decoding.

File: ids/node_id.go (hunk 2 above)
Comment (StephenButtolph) [reply ids/node_id.go:18]: `String()`

**Review by dhrubabasu — APPROVED** [review]

**Pull request merged**